I have completed the first draft of assignment 2.

JenniferOnuora opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
I completed the first draft of assignment 2 for LCR.
## What did you learn from the changes you have made?
I learnt how to plot univariate and multivariate linear regression models, describe associations between predictors and response based on the model and make predictions.
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
 None at the moment.
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
 None at the moment.
## How were these changes tested?
I ran all the codes and ensured that they ran correctly.
## A reference to a related issue in your repository (if applicable)
 None at the moment.
## Checklist
- [ x] I can confirm that my changes are working as intended

JenniferOnuora committed:
I have completed the first draft of assignment 2.

KasraVakiloroayaei reviewed (changes requested):
A2: Incomplete 

Q3: In order to obtain the correct regression coefficients, i would suggest trying an alternative approach to the train, test splitting of the data:

Split the Auto MPG dataset into 75% training data and 25% testing data
mpg_train, mpg_test = train_test_split(mpg_data, train_size=0.75, random_state=42)

Then check the shapes of the training and testing sets to ensure that you have gotten correct split. 

Q4: Recalculate RMSPE... I would suggest using the mean squared error function built into one of the imported packages/libraries we are using:

Calculate the RMSE (Root Mean Squared Error)
rmse = np.sqrt(**mean_squared_error**(test_response, y_pred))

JenniferOnuora committed:
Addressed the feedback for Q3 and Q4

JenniferOnuora committed:
Addressed the Feedback for Q3 and Q4

JenniferOnuora committed:
Addressed the feedback for Q3 and Q4

JenniferOnuora commented:
Please ignore the second and third commits which were made in error (also mistakenly committed old assignment 1 and 3 scripts). I have addressed the feedback for assignment 2 and have submitted as well (check the first commit)

KasraVakiloroayaei reviewed (approved):
A2: Complete 
Good work making the changes
